docs/INTEGRATION_VARIANTS.md

```diff
@@ -0,0 +1,259 @@
+## Variants
+
```

Comment:
I think it would be good to have a short description of what product variants are and why you might want to use them.